scripts/instalar.py

```
@@ -0,0 +1,559 @@
+# Comprobaciones individuales
+# ---------------------------------------------------------------------------
+
+def ejecutar(cmd: list, capture: bool = True, timeout: int | None = 60) -> tuple[bool, str]:
+    """
+    Ejecuta un comando y devuelve (éxito, salida).
+
+    El parámetro `timeout` controla cuántos segundos esperar antes de
+    cancelar el proceso (None = sin límite, útil para instalaciones largas).
+    """
+    try:
+        resultado = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)